### PR TITLE
Add IXA group of the University of the Basque Country

### PR DESCRIPTION
**Pull request by asoroa**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: dblp-aliases.csv [hunk 1]
```diff
@@ -32643,3 +32643,9 @@ Anne Martens,Anne Koziolek
 David Mimno,David M. Mimno
 Chris Bailey,Christopher Bailey 0002
 Jim Austin,James Austin
+Iñaki Alegría Loinaz, Iñaki Alegria
```
Comment (emeryberger) [inline dblp-aliases.csv:32646]: There should be no spaces here or in the entries below.

File: csrankings.csv [hunk 2]
```diff
@@ -15884,3 +15884,32 @@ Peter I. Cowling,University of York,https://www-users.cs.york.ac.uk/~pic/,xQ7IpL
 Delaram Kahrobaei,University of York,https://www-users.cs.york.ac.uk/~delaram/,ScTS9PkAAAAJ
 Jim Austin,University of York,https://www-users.cs.york.ac.uk/~austin/,xpZL8d8AAAAJ
 James Alfred Walker,University of York,https://www.cs.york.ac.uk/people/?group=Research%20Fellows%20and%20Research%20Associates&username=jawalker,Yl5OycsAAAAJ
+Rodrigo Agerri,University of the Basque Country UPV/EHU,www.rodrigoagerri.net,1SJh0a8AAAAJ
```
Comment (emeryberger) [inline csrankings.csv:15887]: According to the commit message, this is only adding a specific group. You need to add the entire faculty if adding a new institution to the database. Also, all of these people need to be professors; this seems like a lot of people just in NLP.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 2 above)
Comment (asoroa) [reply csrankings.csv:15887]: Thanks for the quick reply.
I'vw solved the space issue in the aliases file.
Also, I've removed non tenure-track professors from the list (sorry for including those, I misread the requirements).

**Comment by emeryberger** [comment]
Thanks. Also, this applies (you need to add the entire department, not just one group).

* If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

**Comment by asoroa** [comment]
The people added  belong to the HitZ research center (see http://ixa2.si.ehu.es/hitz). I've added the complete list of proffesors of the center (I should have mentioned it in the description, sorry).

**Comment by emeryberger** [comment]
I have no access to that website. I am still unclear: does this PR now include all CS faculty? Also, does this institution offer a PhD in computer science?

**Comment by asoroa** [comment]
We are making changes in the website, and it temporarily unavailable, sorry. If you want I can notify you when the site is ready. Regarding your questions:

> I am still unclear: does this PR now include all CS faculty? 

The PR includes all the profesors of the HiTZ research center, which belongs to the University of the Basque Country.

> Also, does this institution offer a PhD in computer science?

The Hitz center offers a doctoral program where students can develop a PhD in NLP.

**Comment by emeryberger** [comment]
This does not seem like a CS department per se. Either it's a partial department (which is not allowed; you need the whole faculty to be listed), or it doesn't offer a CS PhD (making it ineligible).

**Comment by emeryberger** [comment]
Closing for lack of response in 2 weeks. Feel free to re-open.

**Pull request closed without merging**